pkgs/by-name/sc/scalpel/package.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+, autoconf
+, automake
```

Comment:
```suggestion
, autoreconfHook
```

Reply:
Tried using `autoreconfHook` , I get some runtime errors on build, and would need to create a patch for `Makefile.am`. Would it be better to rollback and use `autoconf` and `automake` or to create the patch file and use `autoreconfHook` ?